### PR TITLE
Update to new Python versions.

### PR DESCRIPTION
**Pull request by icemac**
Plus Flake8 to make tox happy.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[run]
```
Comment (dataflake) [inline .coveragerc:1]: Can we please drop the hidden (and thus surprising/confusing) `.coveragerc` files and put the configuration in the much more visible `setup.cfg` instead?

**Review by dataflake — APPROVED** [review]
LGTM, but see my note about `.coveragerc`.

Furthermore, it would be nice if the HTML coverage output didn't create yet another toplevel folder that must then be ignored in `.gitignore`. Can we put that in `parts` instead?

File: .coveragerc (hunk 1 above)
Comment (icemac) [reply .coveragerc:1]: Nice idea. I am currently working on a more automatic approach to to make the packages more equal.

**Comment by icemac** [comment]
> Furthermore, it would be nice if the HTML coverage output didn't create yet another toplevel folder that must then be ignored in `.gitignore`. Can we put that in `parts` instead?

There is no `parts` directory before running buildout for the first time, this makes the coverage `tox` environment to fail if it is run before `buildout`.

**Comment by icemac** [comment]
But we could automate `.gitignore`, too.

**Comment by icemac** [comment]
I'll change this in the next PR.

**Comment by dataflake** [comment]
Running `tox` before the buildout has run? I don't understand.

**Comment by dataflake** [comment]
P.S.: OK, I understand where my own disconnect happened: I am still used to the "old times" where the buildout would also give us a `bin/tox`, so running the buildout was the prerequisite for everything. Including running `tox` at all.

I have moved past that in real life, but it still stuck in my mind. But the point is still valid, we should create less rather than more items in the buildout that will then have to be added to the `.gitignore`. That list seems to be growing and growing.

**Pull request merged**